build/make-distribution.sh

```diff
@@ -138,7 +138,7 @@ function build_service {
   # Store the command as an array because $MVN variable might have spaces in it.
   # Normal quoting tricks don't work.
   # See: http://mywiki.wooledge.org/BashFAQ/050
-  BUILD_COMMAND=("$MVN" clean package $MVN_DIST_OPT -pl master,worker,cli -am $@)
+  BUILD_COMMAND=("$MVN" clean package $MVN_DIST_OPT -pl master,worker,cli,verifier -am $@)
```

Comment:
It is better to use profile for `verifier` module, because `make-distribution.sh` is used to make binary file.